### DISCUSSION TITLE
Input type = range customization

### DISCUSSION THREAD

**Discussion by maasir554** [tailwindlabs/tailwindcss#12141] [Ideas]
The options for easy customization of **range input** should also be included. like:
- modify *thumb* styling
- modify *slider-runnable-track* styling
can't we have a simple utility name for these?
like 
```
className = "thumb:bg-green-500 thumb:border-2 slider-bg-green700" 
```

**Comment by lukewarlow** [comment]
These don't have consistent standard pseudo elements yet. So Tailwindcss can't add variants for it. That being said the csswg is working on it so hopefully soon it'll be doable.
